HM_05/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HM_05
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Random rand = new Random();
+            int[] array = new int[3];
+            for (int i = 0; i < array.Length; i++)
+            {
+                array[i] = rand.Next(0, 100);
+            }
+            foreach (var item in array)
+            {
+                Console.WriteLine(item);
+            }
+            int[] array2 = new int[3];
+            for (int i2 = 0; i2 < array2.Length; i2++)
+            {
+              Console.WriteLine("Введите значения для массива 2:");
+                array2[i2] = int.Parse(Console.ReadLine()); 
+            }
+            int[] array3 = new int[3];
```

Comment:
let us imagine the following - we need to do the same for array that has 1000 elements. 
So, in your solution we will have 1000 line of code. It is not flexible approach.